web/src/app/shared/shared.module.ts

```diff
@@ -23,7 +24,8 @@ import { MarkdownDirective } from './directives/markdown.directive';
     DialogConfirmationComponent,
     DialogListComponent,
     ProjectsListComponent,
-    PrivatePublicProjectComponent],
+    PrivatePublicProjectComponent,
+    BreadcrumbComponent],
```

Comment:
Ideally the closing `]` should be on its own line - I realise it was like that before

Reply:
okay done